src/main/java/org/folio/inventory/validation/experimental/TargetItemStatusValidators.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.inventory.validation.experimental;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import java.util.EnumMap;
+
+public class TargetItemStatusValidators {
+  private static final EnumMap<ItemStatusName, AbstractTargetItemStatusValidator> validators = new EnumMap<>(ItemStatusName.class);
+
+  public TargetItemStatusValidators() {
+    validators.put(ItemStatusName.IN_PROCESS, new InProcessTargetItemStatusValidator());
+    validators.put(ItemStatusName.IN_PROCESS_NON_REQUESTABLE, new InProcessNonRequestableTargetItemStatusValidator());
+    validators.put(ItemStatusName.INTELLECTUAL_ITEM, new IntellectualItemTargetItemStatusValidator());
+    validators.put(ItemStatusName.LONG_MISSING, new LongMissingTargetItemStatusValidator());
+  }
+
+  public AbstractTargetItemStatusValidator getValidator(ItemStatusName itemStatusName) {
+    return validators.get(itemStatusName);
+  }
+
```

Comment:
Please remove this unnecessary whitespace as it does not enhance the readability of the code